src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -79,171 +79,139 @@ export default defineConfig({
+:::
 
-**Type:** `'auto' | 'include' | 'exclude'`<br />
-**Default:** `'auto'`
+#### platformProxy.enabled
+<p>
+**Type:** `{ enabled?: boolean }`<br />
+**Default:** `{ enabled: false }`
 </p>
 
-Determines how `routes.json` will be generated if no [custom `_routes.json`](#custom-_routesjson) is provided.
-
-There are three options available:
-
-* **`"auto"` (default):** Will automatically select the strategy that generates the fewest entries. This should almost always be sufficient, so choose this option unless you have a specific reason not to.
+The `enabled` property allows you to enable the Cloudflare runtime in `astro dev`.
 
-* **`include`:** Pages and endpoints that are not pre-rendered are listed as `include` entries, telling Cloudflare to invoke these routes as functions. `exclude` entries are only used to resolve conflicts. Usually the best strategy when your website has mostly static pages and only a few dynamic pages or endpoints.
-
-  Example: For `src/pages/index.astro` (static), `src/pages/company.astro` (static), `src/pages/users/faq.astro` (static) and `/src/pages/users/[id].astro` (SSR) this will produce the following `_routes.json`:
+#### platformProxy.configPath
+<p>
+**Type:** `{ configPath?: string }`<br />
+**Default:** `{ configPath: 'wrangler.toml' }`
+</p>
 
-  ```json title="dist/_routes.json"
-  {
-    "version": 1,
-    "include": [
-      "/_image", // Astro's image endpoint
-      "/users/*" // Dynamic route
-    ],
-    "exclude": [
-      // Static routes that needs to be exempted from the dynamic wildcard route above
-      "/users/faq/",
-      "/users/faq/index.html"
-    ]
-  }
-  ```
+The `configPath` allows you to define your Wrangler configuration file, relative to the root of your Astro project. 
 
-* **`exclude`:** Pre-rendered pages are listed as `exclude` entries (telling Cloudflare to handle these routes as static assets). Usually the best strategy when your website has mostly dynamic pages or endpoints and only a few static pages.
-
-  Example: For the same pages as in the previous example this will produce the following `_routes.json`:
-
-  ```json title="dist/_routes.json"
-  {
-    "version": 1,
-    "include": [
-      "/*" // Handle everything as function except the routes below
-    ],
-    "exclude": [
-      // All static assets
-      "/",
-      "/company/",
-      "/index.html",
-      "/users/faq/",
-      "/favicon.png",
-      "/company/index.html",
-      "/users/faq/index.html"
-    ]
-  }
-  ```
+#### platformProxy.configPath
+<p>
+**Type:** `{ experimentalJsonConfig?: boolean }`<br />
+**Default:** `{ experimentalJsonConfig?: false }`
+</p>
 
-### routes.include
+The `experimentalJsonConfig` property defines whether the utility reads a JSON config file (e.g. `wrangler.json`).
```

Comment:
if for `getPlatformProxy` you set `configPath` to a json file, it will be read no problem

the `experimentalJsonConfig` option makes the utility look for a `wrangler.json`

so you might want to maybe tweak a bit this section 🙂 

(PS: I find this behavior a bit unexpected and confusing but it's the same for `wrangler (pages) dev` and `getPlatformProxy`, also the team finds it appropriate, so it's not an unintentional behavior 😅)